node/pkg/alephium/watcher.go

```diff
@@ -221,6 +236,12 @@ func (w *Watcher) fetchEvents(ctx context.Context, logger *zap.Logger, client *C
 						errC <- err
 						return
 					}
+					if unconfirmed.msg.isAttestTokenVAA() {
+						if err = w.validateAttestToken(ctx, unconfirmed.msg); err != nil {
```

Comment:
The watchers of other chains do the same. When encountering network errors or other errors, the watcher will ignore them, which means that the guardian may miss some vaas. In this case, we need to process the lost vaas through reobserve.

I guess the reason for this is to make the code simpler, it might make the code more complicated if we want to handle network errors and other errors separately.

But I forgot to check the attest token in reobserve, I will fix this in another PR.